### ISSUE TITLE
can't get resource 

### ISSUE THREAD

**Issue by vitqst** [jacwright/RestServer#70]
I can't get resource at link example.com/v1/resource/abc.png 
can you explain me how to fix it or what i forgot when config ? Thank

**Comment by jacwright** [comment]
Should work. Want me to guess what you might have forgotten? ;)

**Comment by tyzoid** [comment]
Feel free to re-open this issue if you can provide a working example of this failing. I suspect the recent fix for #80 may have something to do with this as well, but cannot tell without more information.

**Comment by turbopixel** [comment]
Our rest api doesn't work anymore. Still got a "error: 404" message. The rest api framework can't find the `@url /notifications/getAll` path with the fix 4024098ed93a4

_\Jacwright\RestServer\RestServer::getPath()_ 
Line 368

Before `ltrim()`
 * `notifications/getAll`

After `ltrim()`
 * `/notifications/getAll`

Our system call the rest api on url <domain.tld>/cms-shop/rest/notifications/getAll and received this:

```
{
    "error": {
        "code": 404,
        "message": "Not Found"
    }
}
```

**Comment by tyzoid** [comment]
Is this a regression? If so please open a new bug. In the meantime, I'm looking into it.
